config/jobs/ppc64le-cloud/docker-in-docker/postsubmit-push-docker-in-docker.yaml

```diff
@@ -0,0 +1,98 @@
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.2
+            env:
+              - name: DOCKER_CONFIG
+                valueFrom:
+                  secretKeyRef:
+                    name: quay-powercloud-regcred
+                    key: .dockerconfigjson
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                # Build and install vmware-tanzu/buildkit-cli-for-kubectl
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                # Install manifest-tool
+                curl -L https://github.com/estesp/manifest-tool/releases/download/v1.0.3/manifest-tool-linux-`go env GOARCH` -o /usr/local/bin/manifest-tool
+                chmod +x /usr/local/bin/manifest-tool
+                # Kubeconfig for the x86 cluster
+                ca=$(base64 -w 0 /run/secrets/kubernetes.io/serviceaccount/ca.crt)
+                token=$(cat /run/secrets/kubernetes.io/serviceaccount/token)
+                namespace=$(cat /run/secrets/kubernetes.io/serviceaccount/namespace)
+                cat >kubeconfig <<EOL
+                apiVersion: v1
+                kind: Config
+                clusters:
+                - name: default-cluster
+                  cluster:
+                    certificate-authority-data: ${ca}
+                    server: https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+                contexts:
+                - name: default-context
+                  context:
+                    cluster: default-cluster
+                    namespace: ${namespace}
+                    user: default-user
+                current-context: default-context
+                users:
+                - name: default-user
+                  user:
+                    token: ${token}
+                EOL
+                export KUBECONFIG=$(pwd)/kubeconfig
+                # Get the latest tag in the quay.io/powercloud/docker-ce-build
+                latest_tag=$(curl -H "Authorization: Bearer XYZ" -X GET "https://quay.io/api/v1/repository/powercloud/docker-ce-build/tag/" | jq .tags[].name | grep v | head -1 | sed -e 's|\"||g')
+                latest_major=$(echo $latest_tag | cut -d'v' -f2 | cut -d'.' -f1)
+                latest_minor=$(echo $latest_tag | cut -d'v' -f2 | cut -d'.' -f2)
+                new_tag=$(echo 'v'${latest_major}'.'$((latest_minor+1)))
+                pushd images/docker-in-docker
+                # Build the x86 image
+                kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder -t quay.io/powercloud/docker-ce-build:${new_tag}-amd64 -f Dockerfile ./
```

Comment:
@florencepascual Can you please explain the need for building and having x86 image also in quay.io repo? Is it not available in the official docker repo?
Also, would you be using this x86 image for your job of building docker/containerd packages for ppc64le.

Reply:
We started with an x86 image with a kind cluster. But now we don't use it regularly, since we use a kubeadm ppc64le cluster, on a x86 machine. Although it is possible we may need it to troubleshoot some bugs, it is not necessary. So if you would rather have me focus on a ppc64le image, I can change it.